ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host1/connected_endpoints_keys_key.yml

```diff
@@ -0,0 +1,20 @@
+type: l3leaf
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 192.168.255.0/24
+    vtep_loopback_ipv4_pool: 192.168.254.0/24
+  nodes:
+    - name: connected_endpoints_keys_key
+      id: 101
+      bgp_as: 101
```

Comment:
This config should only be in one file, so remove it from all the others.

Reply:
Done